.github/PULL_REQUEST_TEMPLATE/small-change.md

```diff
@@ -9,6 +9,12 @@
 
 Not applicable.
 
+## Effects on driving behavior
```

Comment:
I'm not sure if it's limited to `driving`. I personally feel `system` is safer.

```suggestion
## Effects on system behavior
```

Reply:
Looks better!